tests/drum/test_custom_model.py

```diff
@@ -178,120 +178,6 @@ def process(self):
 
 
 class TestCMRunner:
```

Comment:
as for refactoring this test class - it's better to have it split into several test files, not several test classes.
having tests in files is more natural for py.test, please it's easier to read and navigate.